codegen/smithy-ruby-rails-codegen/src/main/java/software/amazon/smithy/ruby/codegen/protocol/railsjson/generators/StubsGenerator.java

```diff
@@ -128,6 +130,20 @@ protected void renderPayloadBodyStub(Shape outputShape, MemberShape payloadMembe
         }
     }
 
+    @Override
+    protected void renderHeaderStubbers(Shape shape) {
+        if (shape.hasTrait(ErrorTrait.class)) {
+            RailsJsonTrait railsJsonTrait = context.service().getTrait(RailsJsonTrait.class).get();
+            String errorLocation = railsJsonTrait.getErrorLocation().orElse("status_code");
```

Comment:
How do we handle the status_code case?

Reply:
It's handled already in RestBase by looking at the status code on the error and setting it on the stub.